### PR TITLE
Tyshchenko/ all tests transfer to Page Object

### PR DESCRIPTION
**Pull request by TyshchenkoRoma**

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarURLMozilla.java [hunk 1]
```diff
@@ -99,6 +99,7 @@ public void checkPresentJPGAvatarInEditAvSystChrome() {
     }
 
     @Test
+
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarURLMozilla.java]: remove useless line

File: src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarURLMozilla.java [hunk 2]
```diff
@@ -121,7 +122,6 @@ public void checkPresentPNGAvatarInEditAvSystMozilla() {
         clickOnConfirmButton();
         Assert.assertTrue(getPNGAvatar().isDisplayed());
     }
-
     @Test
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarURLMozilla.java:125]: add line before test

File: src/test/java/rv/atqc/gttesting/archex/EditAvartarPage.java [hunk 3]
```diff
@@ -3,36 +3,93 @@
...
+    @FindBy(xpath = "/html/body/main/div/welcome-page/div[1]/a")
+    private WebElement mainPage;
 
     public EditAvartarPage(WebDriver driver) {
         super(driver);
     }
 
     public String getChangeButtonText() {
-        waitForVisibilityOfElement(chengeButton, MAX_WAIT_TIME);
+        waitForVisibilityOfElement(chengeButton, Integer.valueOf(Resources.LogInConstants.MAX_WAIT_TIME));
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/EditAvartarPage.java]: you can just have const in this class

**Review by dmatsev — APPROVED** [review]
with comments

**Review by romanukhaav — APPROVED** [review]

**Review by CaCO3ua — APPROVED** [review]

**Pull request merged**